index.d.ts

```diff
@@ -48,3 +57,36 @@ console.log('The file has been moved');
 ```
 */
 export function moveFileSync(sourcePath: string, destinationPath: string, options?: Options): void;
+
+/**
+Rename a file asynchronously.
+
+@param source - The file you want to rename.
+@param destination - The name of the renamed file.
+@returns A `Promise` that resolves when the file has been renamed.
+
+@example
+```
+import {renameFile} from 'move-file';
+
+await renameFile('unicorn.png', 'unicorns.png', {cwd: 'source'});
+console.log('The file has been renamed');
+```
+*/
+export function renameFile(source: string, destination: string, options?: Options): Promise<void>;
```

Comment:
```suggestion
export function renameFile(sourceFilename: string, destinationFilename: string, options?: Options): Promise<void>;
```

Just to make it clear it has to be a filename, not a path?

Reply:
I think letting it be a path lets users choose the more ergonomic version for themselves:

```js
await renameFile('../unicorn.txt', '../unicorns.txt');
```

```js
await renameFile('unicorn.txt', 'unicorns.txt', {cwd: path.resolve('some/long/nested/path')});
```

Reply:
👍